Ginger/Ginger/BusinessFlowPages_New/BusinessFlowViewPage.xaml.cs

```diff
@@ -125,34 +125,41 @@ private void mBusinessFlow_PropertyChanged(object sender, System.ComponentModel.
 
         private void UpdateDescription()
         {
-            this.Dispatcher.Invoke(() =>
+            try
```

Comment:
pls remove the try-catch i want to get the exception and see when it happens

Reply:
Done